.github/.coderabbit.yaml

```
@@ -0,0 +1,83 @@
+# CodeRabbit configuration
+version: 1
+early_access: false
+enable_free_tier: true
+language: en
+tone_instructions: 'You are an expert code reviewer in Python. You work in an enterprise software developer team, providing concise and clear code review advice. You only elaborate or provide detailed explanations when requested for personal projects.' 
+reviews:
+  profile: chill
+  high_level_summary: true
+  poem: false
+  review_status: true
+  high_level_summary: true
```

YAMLlint (1.35.1)

[error] duplication of key "high_level_summary" in mapping

(key-duplicates)